packages/io.prometheus/Configuration.pkl

```diff
@@ -284,6 +284,28 @@ class FileSdConfig {
   refresh_interval: Duration?
 }
 
+/// DNS-based service discovery configuration allows specifying a set of DNS domain names which
+/// are periodically queried to discover a list of targets.
+///
+/// https://prometheus.io/docs/prometheus/latest/configuration/configuration/#dns_sd_config
+class DnsSdConfig {
+  /// A list of DNS domain names to be queried.
+  name: Listing<String>
+
+  /// The type of DNS query to perform. One of SRV, A, AAAA, MX or NS.
+  type: *"SRV"|"A"|"AAAA"|"MX"|"NS"
```

Comment:
It's better to stay in line with the target output's schema, and this is more in line with what the type actually is.

For values that have defaults, typically we leave this in as a comment, and set the type to nullable.

```suggestion
  /// The type of DNS query to perform. One of SRV, A, AAAA, MX or NS.
  ///
  /// Default if unspecifioed: `"SRV"`
  type: ("SRV"|"A"|"AAAA"|"MX"|"NS")?
```